automount-actions/scripts/bin/auto-install.sh

```diff
@@ -317,6 +309,11 @@ main() {
     # Globally set exit on error, no unset variables
     set -eu
 
+    : "${LOG_DIR:="$(mktemp -d)"}"; readonly LOG_DIR
+    
+    _log_file="$(mklog)"
+    readonly _log_file
```

Comment:
```suggestion
    "${_log_dir:="$(mktemp -d)"}"; readonly _log_dir
    
    mklog "$_log_dir"; readonly _log_file
```

For uniformity, I guess. The idea behind the naming convention for variables oh so long ago was that you override via eg `environment:` in `snap/snapcraft.yaml` with `VARNAME`, but within the script we work with `_varname`.